advanced/Exercise_3.md

```diff
@@ -0,0 +1,105 @@
+# Exercise 3 - Ignoring files
+
+In this exercise, you will first use a provided script to generate a simple git repository containing materials for planning a conference. You will then create a README file for the conference planning repository using the Markdown language. Then you will generate an HTML README file, and finally create a `.gitignore` file for the repository. This exercise is closely adapted from [this website](https://oesa.pages.ufz.de/git-exercises/exercise-5.html).  
+
+* [Initialize the git repository](#initialize)
+
+* [Create a README file](#readme)
+
+* [Generate the README file in HTML](#html)
+
+* [Create a .gitignore file](#gitignore)
+
+## Initialize the git repository <a name="initialize"></a>
+
+Use the helper scripts provided in https://www.github.com/C2SM/git-course to set up a simple git repository.
+
+First, clone the git-course repository if you have not already done so in Exercise 1.  
+
+```plaintext
+git clone git@github.com:C2SM/git-course
+```
+
+Navigate into the repository.
+
+```plaintext
+cd git-course
+```
+
+Source the file containing the helper scripts: `helpers.sh`
+
+```plaintext
+source helpers.sh
+```
+Run the `init_advanced_repo` script.  This script will create a folder at the same level as the git-course repository containing a simple git repository called `conference_planning`.  
+
+```plaintext
+init_advanced_repo
+```
+
+Take some time to familiarize yourself with the git repository by examining the files and using `git status`, `git log`, `git diff`, etc.
+
+## Create a README file <a name="readme"></a>
+
+Create a README.md file in the `conference_planning` folder.  
+
+```plaintext
+touch README.md
+```
+
+Put some content in the README file using a file editor. Good README file content could include the title of the project, a description of the project, how to use the project, and/or the contributors of the project. The `.md` file extension we have used indicates that the README file will be written using the Markdown language, which is a common format for README files because it displays well in web browsers. [Here](https://www.markdownguide.org/basic-syntax/) are some tips for using the Markdown language.
+
+Now commit the README file to the repository.
+
+```plaintext
+git add README.md;
+git commit -m "Add README file"
+```
+## Generate the README file in HTML <a name="html"></a>
+
+It may be useful to convert the README file into an html file in order to see how it will look to a user web browsing your repository. We will use a document converter called [Pandoc](https://pandoc.org) to do this. You can check if you have Pandoc installed with the following command:
+
+```plaintext
+pandoc -h
+```
+
+If you don't have Pandoc installed, there are instructions on how to do so [here](https://pandoc.org/installing.html).
+
```

Comment:
I hope we don't lose time here if someone has trouble installing pandoc.

Reply:
If someone has an issue installing pandoc, we can simply have them skip the step of creating the HTML file and just make file with the .html extension to see how .gitignore works. I think it's good to keep this in because Pandoc is potentially very useful and they may not have heard about it yet.